Promise.resolve(4 * "asd")
.then((n) =>{
    if (Number.isNaN(n)) {
        throw new Error("Valores invÃ¡lidos")
    }
}).catch((err) => console.log(`Um erro ocorreu: ${err}`))